src/backlight/backlight.c

```diff
@@ -334,6 +359,9 @@ static int run(int argc, char *argv[]) {
 
         log_setup_service();
 
+        if (argc == 2 && streq(argv[1], "--help"))
```

Comment:
Normally we allow help to be present in any position, but I guess we can use this narrow check here: after all, we don't expect people to call the binary too often.

Reply:
On second thought, please change this to:
```C
if (strv_contains(strv_skip(argv, 1), "--help")))
```
It's not much longer but it allows the param to be present in any place.